### DISCUSSION TITLE
I need help using i18next from the client side

### DISCUSSION THREAD

**Discussion by KevinArangu** [i18next/next-i18next#1578] [General]
I am trying to use i18next from the client side, I am following the steps of "Loading Namespaces Dynamically Client Side" in the README, but it does not work for me.

my code is: 
![Captura de pantalla 2021-12-07 133115](https://user-images.githubusercontent.com/38361709/145310342-d7b4baf0-a878-43ad-8b5f-63920af8c9ff.png)

my next-i18next.config:
![Captura de pantalla 2021-12-08 200001](https://user-images.githubusercontent.com/38361709/145310456-a1a31511-5fda-4b84-a707-7d5b73295968.png)

The error in nextjs: 
![Captura de pantalla 2021-12-08 200334](https://user-images.githubusercontent.com/38361709/145310556-40d11cb2-8e00-4069-875d-38073e44b4d6.png)

Thanks.

**Comment by isaachinman** [comment]
And what is the result if you add `i18n` as a dependency to the `useEffect` hook?

**Reply by KevinArangu** [reply]
Exactly the same result.

![hook](https://user-images.githubusercontent.com/38361709/145422482-e65bbf5d-9c6f-420f-bae2-87b3eaeeb710.png)

**Reply by isaachinman** [reply]
The i18n instance will be `null` on first load, but should be initialised very shortly thereafter. Can you provide a reproducible repo?
